src/main/starlark/builtins_bzl/common/java/java_binary_deploy_jar.bzl

```diff
@@ -256,11 +258,12 @@ def _implicit_outputs(binary):
         "unstrippeddeployjar": "%s_deploy.jar.unstripped" % binary_name,
     }
 
-def make_deploy_jars_rule(implementation):
+def make_deploy_jars_rule(implementation, *, create_executable):
```

Comment:
As one might guess, this method has other usages internally, so please set a default value for `create_executable` that preserves the old behavior. Once this change is submitted, I'll update things internally to pass the appropriate value.

Reply:
Done